app/moves/views/dashboard.njk

```diff
@@ -0,0 +1,10 @@
+{% extends "./_layout.njk" %}
+
+{% block headerGridColumnClass %}govuk-grid-column-full view-dashboard{% endblock %}
```

Comment:
I'd drop the `view-dashboard` class.

It's much cleaner to apply modifiers to components using BEM rather than do view specific CSS overrides.